src/components/CompareResults/RevisionRow.tsx

```diff
@@ -322,16 +339,20 @@ function RevisionRow(props: RevisionRowProps) {
             </div>
           </Tooltip>
         </div>
-        <div className='base-value cell' role='cell'>
-          {' '}
-          {formatNumber(baseAvgValue)} {baseUnit}{' '}
+        <div className='browser-name cell' role='cell'>
+          {formatNumber(baseAvgValue)} {baseUnit}
+          {getBrowserDisplay(baseApp, newApp, expanded) && (
+            <span style={{ fontSize: '11.5px' }}>({baseApp})</span>
```

Comment:
We have established font sizes in the styles folder. Import `FontSizeRaw` from styles. And add `...FontSizeRaw.xSmall,` to `.browser-name`. 

And notice how we have to add this twice because we've declared it twice for both light and dark. Another reason to have one general styles for the revisions row.

Reply:
If I apply the font size to `.browser-name` it would affect `{formatNumber(baseAvgValue)} {baseUnit}` and we want the browser name to be smaller

Reply:
I'd have to create 
```
const spanStyle = style({
  fontSize: FontSize.xSmall
});
``` 
because we want the browser name to be smaller than `{formatNumber(baseAvgValue)} {baseUnit}` and not same size

Reply:
It's easier on subtests because it has one style so I can do this 
```
  '.browser-name span': {
          fontSize: FontSize.xSmall,
       }
   ```

Reply:
@Netacci Did you update this? I'm confused